src/boardToJSON.ts

```diff
@@ -0,0 +1,20 @@
+import {GameState} from "./gameState"
+
+export class BoardToJSON{
+    public static convert(board:number[][]):{"ShipPositions":Array<any>,"MyShots": Array<any>,"OpponentsShots": Array<any>} {
```

Comment:
While it seems strange, I don't think it would have been unreasonable to have a container class for this: something like:

```
export class Board {
  private board: number[][];

  constructor(board: number[][]) {
    this.board = board;
  }

  public getResult(tile: string) { // e.g. tile = "A1"
    let row = this.convertRowToIndex(tile.split()[0]);
    let column = tile.split()[1];
    return this.board[row][column]
  }

  //method to turn A -> 1, B -> 2, etc
  private convertRowToIndex(row: string): number { <...> }
}
```

Very quick and dirty above, but do you get the picture?

Additionally, can you see why that might make this method a little more readable?